projects/nx-verdaccio/src/internal/terminal.unit-test.ts

```diff
@@ -66,6 +66,10 @@ describe('objectToCliArgs', () => {
     expect(objectToCliArgs(params)).toStrictEqual([]);
   });
 
+  it('should early return an empty array if params are not present', () => {
```

Comment:
```suggestion
  it('should return an empty array if params are not present', () => {
```

If we can't test that the return is early i would leave it out for brevity.